nixos/modules/services/home-automation/zwave-js.nix

```diff
@@ -0,0 +1,150 @@
+{config, pkgs, lib, ...}:
+
+with lib;
+
+let
+  cfg = config.services.zwave-js;
+  cacheDir = "/var/cache/zwave-js";
+  runtimeDir = "/run/zwave-js";
+  defaultConfig = {
+    storage = {
+      inherit cacheDir;
```

Comment:
Looks like this is only used once, so moving it here instead of inherting it seems like the correct move.